packages/react/src/auto/AutoFormContext.ts

```diff
@@ -12,6 +12,10 @@ export interface AutoFormMetadataContext {
   metadata: ActionMetadata | undefined;
   submit: UseActionFormSubmit<any>;
   submitResult?: AutoFormSubmitResult;
+  model?: {
+    namespace?: string | string[] | null;
+    apiIdentifier?: string;
+  };
```

Comment:
Is this different than the `metadata`? I think metadata there is actually at the model level already

Reply:
The response metadata from useActionMetdata removes the model api id when it goes deeper into the metadata object to get the action metadata. These values from the action are also used as the GQL query variables in `useActionMetadata`